Makefile

```diff
@@ -44,6 +44,26 @@ ifeq ($(TEST_TYPE),coverage)
 		fi; \
 	done
 endif
+ifeq ($(TEST_TYPE),fuzzit)
+	# skip fuzzing for PR
+	if [ "$(TRAVIS_PULL_REQUEST)" = "false" ] || [ "$(FUZZIT_TYPE)" = "local-regression" ] ; then \
```

Comment:
this comes out as `if [ "3093" = "false" ] || [ "fuzzing" = "local-regression" ] ; then \` is that correct?